src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -60,37 +72,42 @@ public boolean doesUserExist(final String userName) {
 
   @Override
   public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
+    updateUser(user, hashedPassword, hashedPassword.isBcrypted() ? "bcrypt_password" : "password");
+  }
+
+  private void updateUser(final DBUser user, final HashedPassword hashedPassword, final String column) {
     Preconditions.checkArgument(user.isValid(), user.getValidationErrorMessage());
 
     try (final Connection con = connectionSupplier.get();
         final PreparedStatement ps =
-            con.prepareStatement("update ta_users set password = ?,  email = ?, admin = ? where username = ?")) {
+            con.prepareStatement("update ta_users set " + column + " = ?,  email = ? where username = ?")) {
```

Comment:
The variable column names are really hard to reason about. I don't see immediately the intent or why, we could use some comments in-code to convey that. Review commentary can be a smell, much of the time that kind of context should be checked in, either in-context with a comment or a design doc.

I suspect this is a good time to step back a bit. Do we really need to support the ability to update either password column? If we are forcing client changes, we would likely spin up a new lobby for those clients to connect to, in that case any lobby code could assume an updated client. That's also an interesting new headache as we now have two instances hitting the same DB, but at least in both cases the code paths are more well known (in the sense that you know legacy always uses md5 and bcrypt will always go through the new code path).